germanium/tools/models.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from .trivials import assert_equal, assert_true, assert_false
+
+
+def get_pks(iterable):
+    return [obj.pk for obj in iterable]
+
+
+def assert_iterable_objs_equal(first, second, msg=None):
+    assert_equal(set(first), set(second), msg)
+
+
+def assert_qs_exists(qs):
+    assert_true(qs.exists())
+
+
+def assert_qs_not_exists(qs):
+    assert_false(qs.exists())
+
+
+def assert_qs_contains_iterable(qs, iterable, msg=None):
+    assert_equal(set(qs.filter(pk__in=get_pks(iterable))), set(iterable), msg)
```

Comment:
Co tuhle metodu pojmenovat jen assert_qs_contains a bude to fungovat jak s iterable, tak s jednotlivým objektem. Kdybych chtěl zkontrolovat, že nějaký konkrétní objekt je v querysetu.